package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV3.java

```diff
@@ -104,6 +104,7 @@ public void upload ( final File file, final String filename ) throws IOException
         }
         catch ( final IOException e )
         {
+            e.printStackTrace ( this.listener.error ( "Failed to create upload archive" ) );
```

Comment:
The e.printStackTrace adds the message form the Exception to the log. Doesn't it?
That's copied form Line 177.

Should I do 
`this.listener.error ( "Failed to create upload archive: " + e.getMessage () );`
instead?